x-pack/plugins/security_solution/common/endpoint/types/index.ts

```diff
@@ -128,6 +128,13 @@ export interface ResolverNode {
   stats: EventStats;
 }
 
+export interface NewResolverTree {
```

Comment:
Could you add a doc comment explaining `NewResolverTree`?

Reply:
@jonathan-buttner added. We can rename it to ResolverTree though. The `New` designation was just to avoid conflicts as we incrementally made these changes.